### PR TITLE
I believe the OAuth is ready

### PR DESCRIPTION
**Pull request by JuanLGC**

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -2,24 +2,56 @@
...
+from oauthlib.oauth2 import WebApplicationClient
+import requests
 from utils import APIException, generate_sitemap, add_user_authentification
 from admin import setup_admin
-from models import db, User
-#from models import Person
+from models import db, User, UserOAuth
+import json
+import os
+import sqlite3
```
Comment (JimenaEB) [inline src/main.py]: Odena los imputs 
https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement

File: src/models.py [hunk 2]
```diff
@@ -84,6 +86,42 @@ def update_user_info(user_id, new_data):
             setattr(user, key, value)
         db.session.commit()
 
+##############################################################################################
```
Comment (JimenaEB) [inline src/models.py]: Tienes que mergear tu clase User Oauth con User

**Review by JimenaEB — CHANGES_REQUESTED** [review]

File: src/main.py (hunk 1 above)
Comment (JuanLGC) [reply src/main.py]: Done

File: src/models.py (hunk 2 above)
Comment (JuanLGC) [reply src/models.py]: Done! y funciona al peluche!

**Pull request merged**